app/javascript/components/subject/CurrentStatus.js

```diff
@@ -50,19 +51,40 @@ class CurrentStatus extends React.Component {
     }
   }
 
+  generateReportEligibility(eligibility) {
+    return (
+      <React.Fragment>
+        {eligibility.eligible && (
+          <span>
+            . This {this.props.isolation ? 'case' : 'monitoree'} is currently eligible to receive a notification today&nbsp;
+            <EligibilityTooltip report_eligibility={this.props.report_eligibility} id={`eltt`} inline={true} />.
+          </span>
+        )}
+        {!eligibility.eligible && (
+          <span>
+            . This {this.props.isolation ? 'case' : 'monitoree'} is not currently eligible to receive a notification today&nbsp;
+            <EligibilityTooltip report_eligibility={this.props.report_eligibility} id={`eltt`} inline={true} />.
+          </span>
+        )}
+      </React.Fragment>
+    );
+  }
+
   render() {
     return (
       <React.Fragment>
         {!this.props.isolation && (
           <h1 className="display-6 pb-3">
-            This monitoree is in the <u>exposure</u> workflow, and their current status is <b>{this.generateStatus(this.props.status)}</b>.
+            This monitoree is in the <u>exposure</u> workflow, and their current status is <b>{this.generateStatus(this.props.status)}</b>
             {this.generateInfoHover(this.props.status)}
+            {this.generateReportEligibility(this.props.report_eligibility)}
           </h1>
         )}
         {this.props.isolation && (
           <h1 className="display-6 pb-3">
-            This monitoree is in the <u>isolation</u> workflow, and their current status is <b>{this.generateStatus(this.props.status)}</b>.
+            This monitoree is in the <u>isolation</u> workflow, and their current status is <b>{this.generateStatus(this.props.status)}</b>
```

Comment:
Rather than generating the status twice, this can be done once in a variable above.

Reply:
Agreed, done.